automation/publish.sh

```diff
@@ -30,6 +30,8 @@ publish_docs() {
     (
         cd /tmp/gh-pages
         git add -A
+        git config user.name "kubevirt-bot"
+        git config user.email "rmohr+kubebot@redhat.com"
```

Comment:
Can we use our own mail here? Or get Roman's ack?

Reply:
We have to use an account not attached to any user I suppose, let me ask him.

Reply:
You can use the email address, it is the "prow" email address. But consider passing it in in the prowjob and don't hardcode it here.

Reply:
Adding env var to the job https://github.com/kubevirt/project-infra/pull/734